src/convertToObject.js

```diff
@@ -3,16 +3,34 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](/convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const startedArray = sourceString
+    .split(';')
+    .map(el => el.trim());
+
+  const arrayWithoutEmptyStrings = startedArray.filter(el => el.length > 0);
```

Comment:
```suggestion
  const startedArray = sourceString
    .split(';')
    .map(el => el.trim())
    .filter(el => el.length > 0);
```
You can avoid creating this variable